controller/array_action/array_mediator_xiv.py

```diff
@@ -233,13 +236,13 @@ def copy_to_existing_volume_from_source(self, name, source_name, source_capacity
             raise array_errors.IllegalObjectName(ex.status)
         except xcli_errors.SourceVolumeBadNameError as ex:
             logger.exception(ex)
-            raise array_errors.ObjectNotFoundError(source_name)
+            raise array_errors.ObjectNotFoundError(source_id)
         except (xcli_errors.VolumeBadNameError, xcli_errors.TargetVolumeBadNameError) as ex:
             logger.exception(ex)
-            raise array_errors.ObjectNotFoundError(name)
+            raise array_errors.ObjectNotFoundError(volume_id)
         except xcli_errors.OperationForbiddenForUserCategoryError as ex:
             logger.exception(ex)
-            raise array_errors.PermissionDeniedError("create volume : {0}".format(name))
+            raise array_errors.PermissionDeniedError("create volume : {0}".format(volume_id))
```

Comment:
I think the original message here is misleading, since it's not really "create"

Reply:
done